CommonData/column.h

```diff
@@ -153,7 +153,7 @@ class Column : public DataSetBaseNode
 			void					labelValueChanged(Label * label, int	anInteger) { labelValueChanged(label, double(anInteger)); }
 			void					labelDisplayChanged(Label * label);
 			
-			bool					setStringValueToRow(		size_t row, const std::string & value,								bool writeToDBAndSetType = true);
+			bool					setStringValue(				size_t row, const std::string & value, const std::string & label = "", bool writeToDB = true); ///< Does two things, if labelButOnlyFromSpreadsheetPaste=="" it will handle user input, as value or label depending on columnType. Otherwise it will simply try to use userEntered as a value. But this will trigger the setting of type
```

Comment:
<img width="834" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/13460dff-942c-47ac-a390-f861ec32be78">
<img width="54" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/e830b329-4ca6-42e8-aad8-e9801c2fa0e5">
<img width="835" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/b41b296c-0630-473b-b9ac-91fa8f0f60e7">

```suggestion
			bool					setStringValue(				size_t row, const std::string & value, const std::string & label = "", bool writeToDB = true); ///< Does two things, if label=="" it will handle user input, as value or label depending on columnType. Otherwise it will simply try to use userEntered as a value. But this will trigger the setting of type
```

Reply:
<img width="225" alt="image" src="https://github.com/jasp-stats/jasp-desktop/assets/29062713/c012fc51-e1e8-452d-b16c-586432de3fa7">